Tests/test_file_jpeg.py

```diff
@@ -445,6 +455,10 @@ def getsampling(im):
         with pytest.raises(TypeError):
             self.roundtrip(hopper(), subsampling="1:1:1")
 
+        # RGB colorspace, no subsampling by default
+        im = self.roundtrip(hopper(), subsampling=3, keep_rgb=True)
```

Comment:
libjpeg doesn't subsample RGB output by default, since it doesn't make sense to have e.g. lower resolution in the green channel.  The test verifies that an RGB image doesn't get subsampled when we use that default.